### PR TITLE
update GraphQL pages

### PR DESCRIPTION
**Pull request by nikolasburk**
Main changes:
- update links to resources
- update explanations about GraphQL server development
- remove usage of Graphcool as BaaS

### CODE REVIEW

File: src/pages/CommunicatingWithGraphQLAPI.js [hunk 1]
```diff
@@ -10,36 +10,40 @@ const content = markdown(markdownOptions)`
 
 In this section, we’ll explain the core concepts you need to know when working with a GraphQL API. 
 
+
+<!--
```
Comment (dabbott) [inline src/pages/CommunicatingWithGraphQLAPI.js]: This TODO block causes a syntax error. Let's either update or remove it?

File: src/pages/GraphQL.js [hunk 2]
```diff
@@ -103,13 +103,15 @@ GraphQL has a rapidly growing community. To stay up-to-date about everything tha
...
+* [GraphQL Europe](https://www.graphql-europe.org/): Europe's biggest GraphQL conference
+* [Prisma blog](https://blog.graph.cool/): Technical deep dives and tutorials all around GraphQL development
 
 For an in-depth learning experience, visit the [How to GraphQL](https://www.howtographql.com/) fullstack tutorial website.
 
 
 ## Minimal Example
 
-Here is a minimal example that demonstrates how to load data using [Graphcool](https://www.graph.cool/) and [Apollo Client](http://dev.apollodata.com/react/). If you want to understand more about how it works, check out the following chapters.
+Here is a minimal example that demonstrates how to load data using [Prisma](https://www.graph.cool/), [\`graphql-yoga\`](https://github.com/graphcool/graphql-yoga) and [Apollo Client](https://www.apollographql.com/client). If you want to understand more about how it works, check out the following chapters.
```
Comment (dabbott) [inline src/pages/GraphQL.js]: Should this Prisma link point to a different website? Looks like it still points to graph.cool

Comment (nikolasburk) [reply src/pages/GraphQL.js]: Ah thanks, great catch! I updated the link to the official Prisma page.

File: src/pages/CommunicatingWithGraphQLAPI.js (hunk 1 above)
Comment (nikolasburk) [reply src/pages/CommunicatingWithGraphQLAPI.js]: I removed it for now, we're planning to update graphql-up soon, so then this section can be updated again :)

**Pull request merged**